content/millennium/r4/financial/financial-transaction.md

```diff
@@ -0,0 +1,206 @@
+
+## Terminology Bindings
+
+<%= terminology_table(:financial_transaction, :r4) %>
+
+## Extensions
+
+* [Financial Transaction Account Number]
+* [Financial Transaction Alias]
+* [Financial Transaction Allocation]
+* [Financial Transaction Amount]
+* [Financial Transaction Card Brand]
+* [Financial Transaction Date]
+* [Financial Transaction Location]
+* [Financial Transaction Method]
+* [Financial Transaction Tendered Amount]
+* [Financial Transaction Type]
+
+### Custom Extensions
+
+All URLs for custom extensions are defined as `https://fhir-ehr.cerner.com/r4/StructureDefinition/{id}`
+
+ ID                                      | Value\[x] Type      | Description
+-----------------------------------------|---------------------|--------------------------------------------------------------------------
+ `financial-transaction-account-number`  | [`string`]          | A value associated to the specific payment method usually represented as the last four digits of a credit card, the check number, the EFT number or Lockbox number.
+ `financial-transaction-alias`           | [`string`]          | Client defined value to represent the combination of the type, subtype, and reason describing the financial transaction.
+ `financial-transaction-allocation`      | [`Extension`]       | Defines how the payment or adjustment is to be allocated across other resources.
+ `financial-transaction-amount`          | [`Money`]           | The total amount of the financial transaction.
+ `financial-transaction-card-brand`      | [`string`]          | Identifies the brand of credit card when credit card is used as a payment method.
+ `financial-transaction-date`            | [`date`]            | Represents the expiration date if method is card, check date if method is check, EFT date if method is EFT or Lockbox date if method is lockbox.
+ `financial-transaction-location`        | [`string`]          | Client configured value representing the location or workflow that the payment was received in.
+ `financial-transaction-method`          | [`string`]          | Describes the method of payment for the financial transaction.
+ `financial-transaction-tendered-amount` | [`Money`]           | The amount of cash originally tendered for payment. This value should be greater than or equal to the amount of the cash payment.
+ `financial-transaction-type`            | [`CodeableConcept`] | The classification of the transaction.
+
+## Create
+
+Create a FinancialTransaction.
+
+    POST /Basic    
+
+_Implementation Notes_
+
+* Only the body fields mentioned below are supported.
+* When integrating your application with a client's production environment, the client will have to provide the appropriate financial-transaction-alias values to send with the financial transaction.
+* The financial-transaction-allocation.target field must reference an Encounter associated with an Account.
```

Comment:
We should document:

- FinancialTransaction and Basic scopes are required

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/4adf16f12daf8a74d8b1caef1c28f22dd3d0b8fd#diff-a2a1182a3c7ddb3de2fa0458e90ddf0cR61

Reply:
4adf16f#diff-a2a1182a3c7ddb3de2fa0458e90ddf0cR61

Reply:
+1